### PR TITLE
Calculate well potentials: check if restart file output is requestd

### PR DESCRIPTION
**Pull request by joakim-hove**
Make sure well potentials are calculated if a restart file is requested.

### CODE REVIEW

**Comment by joakim-hove** [comment]
@GitPaean : Try this?

**Comment by joakim-hove** [comment]
jenkins build this please

File: opm/simulators/wells/BlackoilWellModel_impl.hpp [hunk 1]
```diff
@@ -1084,6 +1084,7 @@ namespace Opm {
 
         const Opm::SummaryConfig& summaryConfig = ebosSimulator_.vanguard().summaryConfig();
         const auto& summaryState = ebosSimulator_.vanguard().summaryState();
+        const bool write_restart_file = ebosSimulator_.vanguard().eclipseState().getRestartConfig().getWriteRestartFile(reportStepIdx);
```
Comment (GitPaean) [inline opm/simulators/wells/BlackoilWellModel_impl.hpp]: I am testing this, while please correct `eclipseState` to be `eclState` .

Comment (joakim-hove) [reply opm/simulators/wells/BlackoilWellModel_impl.hpp]: :+1:

**Comment by joakim-hove** [comment]
jenkins build this please

**Comment by GitPaean** [comment]
It looks like master branch is broken, most likely related to libecl.

**Comment by joakim-hove** [comment]
OK - the failure here is due to the merging of: https://github.com/equinor/libecl/pull/636 - which changes the way binary strings are written to file. The new is correct - I will issue an update data on this.

**Comment by joakim-hove** [comment]
jenkins build this update_data

**Comment by joakim-hove** [comment]
jenkins build this update_data please

**Comment by joakim-hove** [comment]
jenkins build this opm-tests=148 please

**Pull request merged**